### DISCUSSION TITLE
How to resolve name.com checkout issue.

### DISCUSSION THREAD

**Discussion by adityamodz** [community/community#51981] [GitHub Education]
Hello,
I am Aditya Nair, I do have a GitHub student plan, My Github User Name is @adityamodz. When I try to check out the domain in Name.com the amount I have to pay is 0 Rs they are asking for card details during checkout but they suppose to show "This one is on us no need of payment ." but it is showing enter card no. or google pay details I saw this tutorial "https://www.youtube.com/watch?v=5v3d7hPLprw&t=1s" The Boy in this video didn't have to add any details but they are asking me to add.

I know these problem doesn't come under the GitHub problem but I would highly appreciate it if u will inform name.com customer support cause they are taking too long to accept tickets I trust Github Community to Review My Problem.
Thanking you in Anticipation.

Regards,
Aditya Nair.

These are some pictures of my problem:
Moderator Edit: 
I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Comment by samus-aran** [comment]
Hi @adityamodz 

If you are having trouble with redeeming your Global Campus Benefits you can reach out to [GitHub Education support](https://support.github.com/contact/education) and selecting the appropriate option **" I need help with a student developer pack offer "**.

Best of luck!

**Comment by SaarthakMaini** [comment]
Hello @adityamodz ,

I'm facing the exact same issue. If possible, could you please let me know if the issue was resolved? And how can we resolve it? 

Thank You!

**Comment by pwnwriter** [comment]
Hey, facing the same issue here as well. Please get the support team know this issue soon.

**Comment by ghost** [comment]
Same here, with name.com

**Comment by anubhavkrishna1** [comment]
### Reply from name.com team 

name.com team:- I truly appreciate you waiting. You should be able to redeem the promotion. However, for our system to recognize it, you need to add a payment method. It won't be charged as long as the price is set to $0.00, but it's necessary for the transaction to be completed. There's no workaround or way to bypass this, so if you don't have an active payment method, it won't work.  Accepted methods of payment and currencies (https://cs.name.com/hc/en-us/articles/205934897-Accepted-methods-of-payment-and-currencies)

me:- if i had credit card or debit card then why did i go for github education ?

name.com team:- I understand it's frustrating, but unfortunately, that's how the process works. As mentioned before, there’s no workaround or way to bypass this. Without an active payment method, the transaction cannot proceed. It won’t be charged if the price is set to $0.00, but having an active payment method is still required to complete the transaction.

**Comment by amishkey** [comment]
I'm facing the same issue in 2025

**Comment by rakaatmaja** [comment]
I'm having the same problem right now. The only difference is that when the price reached $0 during the login process, name.com wouldn't detect my GitHub account, even though my account is eligible for this offer. Has anyone else experienced the same thing?

**Reply by ZiglaCity** [reply]
bro i've literally been facing the same issue, i thought name.com and namecheap.com were the same.... apparently we can only get a .me from namecheap.com, buh name.com keeps acting stubborn. Its annoying!

**Reply by rakaatmaja** [reply]
I'm also confused, bro. I've contacted support, but the problem with name.com hasn't been resolved. It's really confusing.

I've tried to get a domain from namecheap, but my university email isn't eligible, which is disappointing.

**Reply by Sudip-2** [reply]
yes me too

**Reply by hamzah1432** [reply]
If you are stuck in the login loop or the price reverts to full price at checkout, check your Contact Information profile on Name.com.

The issue for me was the Zip/Postal Code field. I had entered text (my city name) instead of a valid numeric code. The system fails silently at checkout if the address format is invalid.

The Fix:

Log in to Name.com.

Go to Account Settings

Ensure your address is real and, most importantly, change the Zip/Postal Code to a valid number (not text).

Save, then try

Log in WITH GOOGLE
Go to https://www.name.com/partner/github-students MAKE SURE YOU ARE LOGGED IN
Get a domain w/ any other perks
Redeem it (YOU WILL NEED A DEBIT CARD)
and if i remembered the steps right, it should work!

**Comment by redyf** [comment]
Can confirm the same issue happened to me, still trying to find a solution for it.

**Comment by data-spice** [comment]
Hi @adityamodz 👋,

Thanks for sharing your issue! Sometimes Name.com may still ask for payment details (like a card or Google Pay) even when the total is ₹0. This is usually for verification or fraud prevention — they don’t actually charge your card.

A few things you can try:

Make sure you’re accessing the Name.com offer through the GitHub Student Developer Pack page, not directly.

Try using a different browser or clearing cookies before checkout.

If the message still appears, it’s best to contact Name.com support directly, since GitHub doesn’t manage their billing or verification process.
